src/patternfly/components/Breadcrumb/examples/Breadcrumb.md

```diff
@@ -12,21 +12,21 @@ cssPrefix: pf-c-breadcrumb
       {{#> breadcrumb-link}}
         Section home
       {{/breadcrumb-link}}
-      {{#> breadcrumb-item-divider}}{{/breadcrumb-item-divider}}
     {{/breadcrumb-item}}
     {{#> breadcrumb-item}}
+      {{#> breadcrumb-item-divider}}{{/breadcrumb-item-divider}}
```

Comment:
You can use `{{#> breadcrumb-item-divider}}`

Reply:
updated